content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md

```diff
@@ -142,11 +142,11 @@ There are three main types of task suitable to run as a Job:
 1. Non-parallel Jobs
   - normally, only one Pod is started, unless the Pod fails.
   - the Job is complete as soon as its Pod terminates successfully.
-1. Parallel Jobs with a *fixed completion count*:
+2. Parallel Jobs with a *fixed completion count*:
   - specify a non-zero positive value for `.spec.completions`.
   - the Job represents the overall task, and is complete when there is one successful Pod for each value in the range 1 to `.spec.completions`.
   - **not implemented yet:** Each Pod is passed a different index in the range 1 to `.spec.completions`.
-1. Parallel Jobs with a *work queue*:
+3. Parallel Jobs with a *work queue*:
```

Comment:
It seems to me that the original numbering is valid Markdown, but the bullets aren't being parsed right.
The reason for leaving all the numbers as `1.` is that it makes future editing easier as list items are removed or added.


My suggestion for how this part of the page should look as Markdown source:
```markdown
1. Non-parallel Jobs
   * normally, only one Pod is started, unless the Pod fails.
   * the Job is complete as soon as its Pod terminates successfully.
1. Parallel Jobs with a *fixed completion count*:
   * specify a non-zero positive value for `.spec.completions`.
   * the Job represents the overall task, and is complete when there is one successful Pod for each value in the range 1 to `.spec.completions`.
   * **not implemented yet:** Each Pod is passed a different index in the range 1 to `.spec.completions`.
1. Parallel Jobs with a *work queue*:
   * do not specify `.spec.completions`, default to `.spec.parallelism`.
   * the Pods must coordinate amongst themselves or an external service to determine what each should work on. For example, a Pod might fetch a batch of up to N items from the work queue.
   * each Pod is independently capable of determining whether or not all its peers are done, and thus that the entire Job is done.
   * when _any_ Pod from the Job terminates with success, no new Pods are created.
   * once at least one Pod has terminated with success and all Pods are terminated, then the Job is completed with success.
   * once any Pod has exited with success, no other Pod should still be doing any work for this task or writing any output.  They should all be in the process of exiting.
```

What do you think about that approach, @borod108 ?

Reply:
@sftim than you for your quick review! How will the end user see it then?

Reply:
You can [preview your changes locally](https://kubernetes.io/docs/contribute/new-content/new-content/#preview-locally), or push them and view the Netlify preview:
https://deploy-preview-21177--kubernetes-io-master-staging.netlify.app/docs/concepts/workloads/controllers/jobs-run-to-completion/

Do either of those help?

Reply:
@borod108 did you get time to preview these changes?